src/components/TransferOrderItem.vue

```diff
@@ -100,6 +100,12 @@ export default defineComponent({
     }),
   },
   methods: {
+    setProgressBarColor(event:any, item: any) {
```

Comment:
No need to define event as its not used anywhere in the method.

Reply:
Done